### PR TITLE
[EAP7-1916] Add ee-security quickstart to the EAP quickstarts repo

### PR DESCRIPTION
**Pull request by fjuma**
https://issues.redhat.com/browse/EAP7-1916

### CODE REVIEW

File: security/EAP7-1916-add-ee-security-to-EAP-quickstarts.adoc [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+////
+Draft verbiage for up to a few sentences on the feature for inclusion in the
+Release Note blog article for the release that first includes this feature. 
+Example article: http://wildfly.org/news/2018/08/30/WildFly14-Final-Released/.
+This content will be edited, so there is no need to make it perfect or discuss
+what release it appears in.  "See Overview" is acceptable if the overview is
+suitable. For simple features best covered as an item in a bullet-point list 
+of features containing a few words on each, use "Bullet point: <The few words>" 
+////
+A new quickstart that demonstrates how to configure EE security has been added to the EAP quickstarts.
```
Comment (emmartins) [inline security/EAP7-1916-add-ee-security-to-EAP-quickstarts.adoc:107]: A new ee-security quickstart...

File: security/EAP7-1916-add-ee-security-to-EAP-quickstarts.adoc [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+== Security Considerations
+
+////
+Identification if any security implications that may need to be considered with this feature
+or a confirmation that there are no security implications to consider.
+////
+
+== Test Plan
+
+== Community Documentation
```
Comment (emmartins) [inline security/EAP7-1916-add-ee-security-to-EAP-quickstarts.adoc:93]: The quickstart and it's documentation is already in the community.

File: security/EAP7-1916-add-ee-security-to-EAP-quickstarts.adoc (hunk 1 above)
Comment (fjuma) [reply security/EAP7-1916-add-ee-security-to-EAP-quickstarts.adoc]: Done.

File: security/EAP7-1916-add-ee-security-to-EAP-quickstarts.adoc (hunk 2 above)
Comment (fjuma) [reply security/EAP7-1916-add-ee-security-to-EAP-quickstarts.adoc:93]: Done.

File: quickstarts/EAP7-1916-add-ee-security-to-EAP-quickstarts.adoc [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+////
+
+== Security Considerations
+
+////
+Identification if any security implications that may need to be considered with this feature
+or a confirmation that there are no security implications to consider.
+////
+
+== Test Plan
```
Comment (OndrejKotek) [inline quickstarts/EAP7-1916-add-ee-security-to-EAP-quickstarts.adoc:92]: What is the expected scope of testing on the engineering side? Just manual verification? Do you plan to add some (unit) tests to the project?

File: quickstarts/EAP7-1916-add-ee-security-to-EAP-quickstarts.adoc [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+* [x] Managed domain
+
+* [x] OpenShift s2i
+
+* [x] Bootable jar
+
+== Requirements
+
+=== Hard Requirements
```
Comment (OndrejKotek) [inline quickstarts/EAP7-1916-add-ee-security-to-EAP-quickstarts.adoc:53]: Is update to Jakarta APIs in scope of this RFE?

File: quickstarts/EAP7-1916-add-ee-security-to-EAP-quickstarts.adoc (hunk 3 above)
Comment (fjuma) [reply quickstarts/EAP7-1916-add-ee-security-to-EAP-quickstarts.adoc:92]: Manual verification will be done. I've updated the analysis to reflect this.

File: quickstarts/EAP7-1916-add-ee-security-to-EAP-quickstarts.adoc (hunk 4 above)
Comment (fjuma) [reply quickstarts/EAP7-1916-add-ee-security-to-EAP-quickstarts.adoc:53]: @OndrejKotek Confirmed with @emmartins today that updates to Jakarta namespace will be handled as part of a separate RFE.

File: quickstarts/EAP7-1916-add-ee-security-to-EAP-quickstarts.adoc [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+// Discuss with QE during the Kickoff state to decide this
+* [ ] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components
+
+=== Other Interested Projects
+
+=== Relevant Installation Types
```
Comment (kstekovi) [inline quickstarts/EAP7-1916-add-ee-security-to-EAP-quickstarts.adoc:40]: In README of quickstart is isn't mentioned managed domain, bootable jar. And OpenShift is explicitly mentioned as not compatible for this QS.

Should be update the AD or README?

**Comment by bstansberry** [comment]
@emmartins I'm going to close this instead of merging as it's not about WildFly.

**Pull request closed without merging**